server/game/cards/events/02/trialbycombat.js

```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../../drawcard.js');
+const ApplyClaim = require('../../../gamesteps/challenge/applyclaim.js');
+
+class TrialByCombat extends DrawCard {
+	setupCardAbilities(ability) {
+		this.interrupt({
+            when: {
+                onClaimApplied: (event, challenge) => (
+                    challenge.winner === this.controller &&
+                    challenge.attackingPlayer === this.controller &&
+                    challenge.challengeType === 'intrigue'
+                )
+            },
+            handler: context => {
+                let opponent = this.game.getOtherPlayer(this.controller);
+
+                context.skipHandler();
+
+                this.game.addMessage('{0} uses {1} to have {2} apply military claim instead of intrigue claim', this.controller, this, opponent);
+
```

Comment:
If you do:
this.game.addMessage('{0} uses {1} to have {2} apply {3} claim instead of {4} claim', this.controller, this, opponent, 'military', 'intrigue');

instead you'll get pretty pictures in chat